rules/import-order.js

```diff
@@ -85,7 +85,23 @@ function isInVariableDeclarator(node) {
 module.exports = function importOrderRule(context) {
   var imported = [];
   var options = context.options[0] || {};
-  var order = options.order || defaultOrder;
+
+  var order = (options.order || []).slice();
```

Comment:
`options.order` comes in as a frozen array. The `slice()` call gets us a mutable clone.